content/en/docs/Tasks/update-operator-to-next-version.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Update operator to next version"
+date: 2021-03-09
+weight: 2
+description: >
+   Given an operator at version n, what should I do to create a version n+1.
+--- 
+
+## Introduction
+
+In the Operator Lifecycle Manager (OLM) ecosystem, the following resources are used to resolve Operator installations and upgrades:
```

Comment:
I do not think that we need to define the resources here. I'd instead of that create links for where they are defined in the docs already. Also, I'd not use `In the Operator Lifecycle Manager (OLM) ecosystem` because the user is in the OLM docs already. 

Then, here I'd put something such as; The following guide will describe how this process works manually and also how [operator sdk][link] CLI tool can help you achieve this goal.